### PR TITLE
Update index.yml

### PR DESCRIPTION
**Pull request by ABDruid**

### CODE REVIEW

**Review by jakerockland — APPROVED** [review]

File: powered-by-art-blocks-pbab-onboarding/index.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
 icon: plug
-label: Powered by Art Blocks (PBAB) Onboarding
+label: Art Blocks Engine Onboarding
 expanded: true
-route: /creator-docs/powered-by-art-blocks-pbab-onboarding/
\ No newline at end of file
+route: /creator-docs/Art-Blocks-Engine-onboarding/
```
Comment (jakerockland) [inline powered-by-art-blocks-pbab-onboarding/index.yml:4]: I'd suggest that this should probably be all lowercase.

**Review by jakerockland — APPROVED** [review]

File: powered-by-art-blocks-pbab-onboarding/index.yml (hunk 1 above)
Comment (aaronpenne) [reply powered-by-art-blocks-pbab-onboarding/index.yml:4]: +1

Comment (jakerockland) [reply powered-by-art-blocks-pbab-onboarding/index.yml]: I made the update

Comment (jakerockland) [reply powered-by-art-blocks-pbab-onboarding/index.yml]: :)

**Comment by jakerockland** [comment]
Ah actually this was done in a different PR–can just close this.

**Pull request closed without merging**